usecases/classification/validation.go

```diff
@@ -27,17 +27,16 @@ const (
 )
 
 type Validator struct {
-	schema  schema.Schema
-	errors  *errorcompounder.SafeErrorCompounder
-	subject models.Classification
+	getClass func(string) *models.Class
+	errors   *errorcompounder.SafeErrorCompounder
+	subject  models.Classification
 }
 
 func NewValidator(sg schemaUC.SchemaGetter, subject models.Classification) *Validator {
```

Comment:
**suggestion:**  i think we don't need the whole `sg schemaUC.SchemaGetter` interface we could directly pass the get func to avoid passing nil